rslib/src/storage/sqlite.rs

```diff
@@ -374,10 +375,11 @@ fn add_extract_fsrs_relative_retrievability(db: &Connection) -> rusqlite::Result
                     {
                         // avoid div by zero
                         desired_retrievability = desired_retrievability.max(0.0001);
+                        let decay = card_data.decay.unwrap_or(0.5);
```

Comment:
You've used 0.2 and 0.5 literals in quite a few places in these changes. It would be nicer if you split them into constants, and used the constants instead of the inline magic values.

Reply:
I added them to FSRS-rs crate.